lib/shipcloud/shipment_quote.rb

```diff
@@ -0,0 +1,24 @@
+module Shipcloud
+  class ShipmentQuote < Base
+    include Shipcloud::Operations::Create
+
+    attr_accessor :from, :to, :carrier, :package, :service
+    attr_reader :shipment_quote
+
+    def self.base_url
+      "#{camel_to_snakecase(class_name)}s"
+    end
+
+    def self.price
+      shipment_quote.price
+    end
+
+    def self.camel_to_snakecase(string)
+      string.gsub(/::/, '/').
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.